src/shogun/statistical_testing/MMD.cpp

```diff
@@ -136,6 +139,16 @@ index_t CMMD::get_num_null_samples() const
 	return self->num_null_samples;
 }
 
+void CMMD::set_random_seed(int32_t p_seed)
+{
+	self->seed = p_seed;
+}
+
+index_t CMMD::get_random_seed() const
+{
+	return self->seed;
+}
+
 void CMMD::set_statistic_type(EStatisticType stype)
```

Comment:
* none of the links work
 * kernel width has to be chosen to something. 1 would be a better choice than 0.25 I agree arbitrary values are bad style
 * I don't like the default seed parameter at all
 * also, why not let the seed be arbitrary when not specified? And then some of the logic here has to be rewritten to work with a local rng. @lambday can you advide here?